### PR TITLE
Added postgresdb install and configured into kruize.

### PR DESCRIPTION
**Pull request by msvinaykumar**

### CODE REVIEW

**Comment by dinogun** [comment]
@msvinaykumar please resolve the conflicts

File: manifests/crc/default-db-included-installation/minikube/kruize-crc-minikube.yaml [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    metadata:
+      labels:
+        app: postgres
+    spec:
+      #securityContext:      #todo
+      #  runAsNonRoot: true
+      #  runAsUser: 1000
+      containers:
+        - name: postgres
+          image: postgres:latest
```
Comment (dinogun) [inline manifests/crc/default-db-included-installation/minikube/kruize-crc-minikube.yaml:54]: In general it is not a good practice to use the `latest` tag. Please use `15.2` as that seems to be the current `latest`

File: manifests/crc/default-db-included-installation/opensshift/kruize-crc-openshift.yaml [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+      #  runAsNonRoot: true
+      #  runAsUser: 1000
+      #serviceAccountName: postgres-sa
+      #securityContext:
+      #  runAsUser: 1001
+      #  runAsGroup: 1001
+      #  fsGroup: 1001
+      containers:
+        - name: postgres
+          image: postgres:latest
```
Comment (dinogun) [inline manifests/crc/default-db-included-installation/opensshift/kruize-crc-openshift.yaml:65]: Same here

**Comment by chandrams** [comment]
@msvinaykumar - I was looking at the pod describe log and I see a warning

 Warning  Unhealthy  43s (x9 over 103s)  kubelet            Liveness probe failed: chmod: cannot access '/home/autotune/app/target/bin/TestDBConnection': No such file or directory

We will need to update the workflows to validate the working of liveness probe.

**Comment by chandrams** [comment]
@msvinaykumar - Please update the deploy script too as it references the crc yamls and the path is now changed.

File: manifests/crc/BYODB-installation/minikube/kruize-crc-minikube.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      name: kruize
+  template:
+    metadata:
+      labels:
+        app: kruize
+        name: kruize
+    spec:
+      containers:
+        - name: kruize
+          image: kruize/autotune_operator:0.0.8_mvp
```
Comment (chandrams) [inline manifests/crc/BYODB-installation/minikube/kruize-crc-minikube.yaml:26]: Please update the docker image tag

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**